pinakes/main/analytics/collector.py

```diff
@@ -0,0 +1,83 @@
+from django.conf import settings
+from django.db import connection
+
+from rest_framework.fields import DateTimeField
+from rq import get_current_job
+
+from insights_analytics_collector import Collector
+from pinakes.main.analytics.package import Package
+
+
+class AnalyticsCollector(Collector):
+    @staticmethod
+    def db_connection():
+        return connection
+
+    @staticmethod
+    def _package_class():
+        return Package
+
+    def get_last_gathering(self):
+        return self._last_gathering()
+
+    def _is_shipping_configured(self):
+        if not settings.PINAKES_INSIGHTS_TRACKING_STATE:
+            self.logger.error(
+                "Insights for Automation Service Catalog is not enabled."
+            )
+            return False
+
+        if not settings.PINAKES_INSIGHTS_URL:
+            self.logger.error("AUTOMATION_ANALYTICS_URL is not set")
```

Comment:
@hsong-rh This message doesn't match the env var that we are checking, this should report on PINAKES_INSIGHTS_URL not being set